### DISCUSSION TITLE
if entity attribute matches entity

### DISCUSSION THREAD

**Discussion by mattp84** [ExperienceLovelace/ha-floorplan#445] [Q&A]
Hi,
Just starting off with floorplan and I'm not sure if this is possible or not but i would like my alarm svg element to be visible when
1. Alarm is not disabled AND a sensor entity is triggered
or
2. The alarm entity attribute that displays the entity that triggered the alarm matches the entity/element combo
or
3. Theres another way my brain cant figure out

if anyone could point in the direction they would go to figure this out as i cant seem to find anything that match what i'm trying to do

your help is greatly appreciated,

Matt

**Comment by OzGav** [comment]
I would use a template to sort this out
